src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -129,7 +129,7 @@ import { ApplicantAddressCopyPage } from '../../shared/components/applicantLists
 import { hasReq } from '../../shared/components/fileUploads/MissingFileOverview';
 import { fileWithMetadataSchema } from '../../shared/components/fileUploads/attachments';
 
-// import mockData from '../tests/fixtures/data/test-data.json';
+// import mockData from '../tests/e2e/fixtures/data/test-data.json';
```

Comment:
This test-data.json file was relocated. This import is updated here and in several other places.